### ISSUE TITLE
Tomatoes SaaS pricing

### ISSUE THREAD

**Issue by potomak** [tomatoes-app/tomatoes#246]
#### Introduction

Tomatoes has started more than 5 years ago as a side project. At the beginning it only served two main purposes:

1. it was an excuse to learn more about Mongodb
1. it was a tool that helped me to stay focused during the first period of my new professional role as a freelancer

The first purpose has been fulfilled. I learnt a lot about Mongodb and Mongoid along the way, but in the end I must say that Mongodb has been a wrong choice from an architectural perspective. It's the wrong tool for the job. It's been *misused* to map a highly relational model on a document store and now I feel it's *too late*, in terms of effort, to replace it with a relational db.

The second purpose has been fulfilled as well. I started using Tomatoes and I never stopped. The only downside is that the tool doesn't promote a strictly application of the Pomodoro Technique®, but I really liked the fact that I was able to *track the time* I spent on different projects and, at the same time, I was able to apply a framework that *helped me stay focused* on one task at a time.

#### Open source software

I started posting about the tool and I received encouraging feedback. Other people from around the world were using it. At the beginning it was rewarding.

I decided to open source the project because I hoped to find collaborators and I wanted to let other people know about my work. I didn't think it would have never been a source of income and I never planned to spend a lot of time developing the idea further, so I thought that releasing the code would have helped keeping the project alive by letting other developers fix bugs and add features.

This choice has been a success from a personal point of view, but a complete failure from a project perspective.

Almost all of the the people that requested new features or bug fixing never actively helped by writing any code. Nobody has never even tried to host their own version of the app from what I know.

#### Project sustainability

When I started displaying ads to pay for app and db hosting I received numerous complains.

Only lately, when I started asking *hardcore Tomatoes users* for help, by supporting the project through [Backerpass](https://backerpass.com/tomatoes), I received a strong positive signal. There was people willing to pay to continue using the hosted version of the project.

I'm very grateful to these **6 heroic users**, excluding me, that are willing to pay to use a tool that they consider useful, but **I don't consider fair that 7 people pay to keep alive a project that is being used daily by at least 100/150 people**.

That's the main reason why **I'd like to make http://www.tomato.es a paid only service**.

#### SaaS

This proposal includes a plan, and promotes discussion, on the topic of making the official hosted version of the project a paid only service.

People who are not willing to pay could either stop using Tomatoes, host their own version of the service, or use a unofficial hosted version of the service.

#### Pricing plans

I have two pricing plans in mind:

1. 24 $/year
2. 4 $/month

The reason why the yearly plan costs 50% less than the monthly plan, is that there's a fixed transaction fee that would highly impact on smaller transactions and because this would promote a longer commitment.

#### Free plan

**There wouldn't be any free plan.**

In my opinion if you *need* a tool or a service, you'd be willing to pay for it. The key concept here is *need*: it means that you're not able to find any replacement that is good enough to make the transition.

Already registered users that are not willing to pay will lose all of their information after a *grace period*. During this transition period users will be able to download all of their data in a CSV format. After that period user accounts, projects, tracked tomatoes, and all the associated information, will be removed from the database permanently. This will help keeping service's operational costs at a minimum.

#### Current supporters

Right now supporters commitment is:

1. Ben: $9
1. Bud Manz: $7
1. Jake Snell: $7
1. johnkrane: $4
1. Giovanni Cappellotto: $3
1. Gabe Ragland: $3
1. Anna C.: $1

The average monthly contribution is around 5 $/month.

I'd like to offer these users a *special offer*: 12 $/year or 2 $/month. For these users the offer will be available forever.

#### Free trial

New users will have a **free evaluation period of 1 month** when they'll be able to try the app.

At the end of the free month they'll start paying the monthly service fee if they don't explicitly opt-out or choose to enroll in the yearly plan.

This means that new users will be asked to fill in their credit card information upon registration.

This will lead to two main effects:

1. new users conversion will drop drastically
1. users that forgot to opt-out will start to pay even if they don't like the service

(1) isn't negative because I'd care *only* about people that are willing to pay. This would potentially exclude a lot of people that could find the service useful, but that are not incline to give their credit card info only to try a service.

(2) is very negative in my opinion because it happened to me the same a couple of times and I felt like I was being deceived to pay for a service that I didn't care about.

Sending *notifications* to warn a user about the imminent end of the free trial period, would help alleviate (2), but this means adding an *email field* to the registration form that would impact negatively on new users conversion.

#### Alternative free trial

An alternative solution would be to ask for credit card information only after the free trial period.

I feel this solution would be slightly harder from an implementation perspective.

It wouldn't impact at all on new users conversion, but it wouldn't prevent registration of users that don't care about the service enough to even fill in their credit card information.

Another positive outcome of this strategy: users that forget to out-out won't *accidentally* pay for a service they don't care about.

**Comment by yoX64** [comment]
Here are my 2 cents, if you don't mind.

I am not sure that the transformation to SaaS will be a good thing. I believe that most of the people use the service because it's free and very easy to use.

When you start charging people for this service, the whole motivation and opinions of people change, because they pay. People that pay a service have the sense that they somehow own that thing. Even if it's only a feeling, it's a strong feeling that motivates people to pay or not to pay a service. This is not a bad thing, but I believe people will be pickier because of this. In order to satisfy the customers or to gather more, a lot development will probably be needed for new features and improvements. Do you have the time to invest?

Another thing is that you are motivated to change to SaaS because you don't think that it's fair that 7 people maintain the project for 100 users. It seems to be unfair. But, remember that because this is an open source project, people are volunteering to maintain the project and they can leave whenever they want, that's the beauty. Considering this, I don't think it's unfair, as long as they want to do this and they choose to contribute. There are thousands of projects in this situation and it's okay, because this means open source.

Bottom line:
 - move to SaaS if you have the time to invest in development.
 - don't be surprised if a lot of people will stop using the service.
 - SaaS vs open source it's your choice, it does not depend so much on others.
 - I recommend open source, because people will be more happy to use it this way.

Anyway, you've done a great job so far and encourage you to continue!

**Comment by dmahlow** [comment]
I'd pay maybe $1 per month for it, but 4$, that's like half a Netflix subscription. I like the service, but there are many alternatives, especially if you don't take advantage of the history and leadboard stuff. Just my 2 cents, I appreciate the service and use it daily, but at the end of the day, there is such a selection of free pomodoro apps, that it's essentially a commodity and from a paying consumer perspective I would treat it as such.

**Comment by potomak** [comment]
> Here are my 2 cents, if you don't mind.

@yoX64 I don't mind at all, on the contrary, thanks for starting the discussion!

> I believe people will be pickier because of this. In order to satisfy the customers or to gather more, a lot development will probably be needed for new features and improvements. Do you have the time to invest?

This is a very interesting question. The answer is "I believe so", but what's more important is that people will decree this by sticking to the app or vice versa.

> Another thing is that you are motivated to change to SaaS because you don't think that it's fair that 7 people maintain the project for 100 users. It seems to be unfair. But, remember that because this is an open source project, people are volunteering to maintain the project and they can leave whenever they want, that's the beauty.

I think there's some confusion here. Tomatoes, as in all the code that you can find at https://github.com/tomatoes-app/tomatoes, is and will be an open source app. This won't change.

What I'd like to change is the pricing strategy of http://www.tomato.es, that is the official hosting of the open source app. Now there's no pricing strategy at all, I'd like to change it by introducing one.

**Comment by potomak** [comment]
@dmahlow in general I don't agree with you on pricing plans, but I agree with you that many people won't be willing to pay that amount to use the app.

You'd be able to pay $1 per month though.

You could join the people who already support Tomatoes at https://backerpass.com/tomatoes to get access to the _special offer_. This way you could pay a yearly subscription $12.

**Comment by Cobradabest** [comment]
I'm going to be honest, I think it's a terrible idea. There are tonnes of free pomodoro apps and websites, and I don't think anybody is going to pay for something they can get elsewhere for free... Paying for a premium account to have no ads and just have free users see ads is one thing, but making the entire service pay only is something else...

**Comment by algor1th** [comment]
To be honest, i chose tomato.es because it was easy to use, and worked on any device. I rarely leave the main page, the features i use are the clock, and the stats shown below that. And (as biased as it is) i think those features should be, and stay free. I am sure you have spent much thought on the statistics features and all the data you collect. And it would be fair that you want to charge people for that work, and the hosting costs you have. 
But i also believe that the core features of tomato.es should be free. If you have features that provide value to users they can not get anywhere else, it makes sense that you charge for that. But the website should still be usable without making an account or spending any money. For example you could calculate at what times people are most productive and help them focus their task priorities around that. But making the service paid only will send many people(like me) who are there just for a timer away and give them the hassle of looking for a new pomodoro timer. Just my two cents

**Comment by zenkamal** [comment]
Here's a perspective from someone (myself!) who previously backed the project using Backerpass, but then stopped.

I was willing to support the project financially, but a couple things made me cancel my Backerpass donation:

-Personally, paying for it actually made me *less* likely to use the service after awhile. It seems (very) counter-intuitive, I know. But I enjoy using both free services (like the various google apps) and paid services, but for paid services I have a really high bar for quality. So since the leaderboards weren't working after a couple weeks, I cancelled my donation and stopped using the service until the leaderboards started working again. For free services, I'm more willing to stick with them through thick and thin. Paying for the service also added some stress to tracking, so I just ditched tracking for a while. I may be unusual in this response though.

-I'm probably going to donate again though, since the leaderboards are working now. 

-Part of the appeal of tomato.es for me is being on the first page or two of the leaderboard. If there are only paying users (which I imagine won't be huge in number), it makes it less appealing to be #15, since that's probably out of 30 total people or fewer, rather than 200 or more.

**Comment by andrew-chiz-muzychuk** [comment]
As a Software Engineer myself, I would want to be paid for my work. As a user I don't want to pay :-)

What I can suggest is make ticking feature free and all the statistics paid. I use statistics for my work. I actually bill my clients basing on your app. So actually, paying $24 a year, when I earn $30 in an hour is totally fair.
But the feeling that it was free and now the owner wants $ for it makes me sad.
Another thing that stops me from accepting paying for tomato.es is that don't have credit card, but only bank account. Paying for the service is really painful for me.

**Comment by okkes** [comment]
I use the app mainly because it's simple, shows how many Poms I did in the past and it was compatible with Habitica. Making this a paid service will make me stop using the app for sure. There are too many other free apps that satisfy my need.

Like other peeps said, adding new functionality and asking a fee for that one is fine. Turning something that was free for a long time into a paid service feels just wrong. (kugh ps4 online). If the server/hosting costs are getting too much, you may want to add advertisements to your web, perhaps after every 4 poms (15 mins break). Just make sure to announce somewhere on your page to disable Adblock and I will fully support that.

**Comment by thereallouiew** [comment]
I think its a great idea but to be more competitive against other free apps, I want to see:

1.  More stats about my work! Maybe an option to see how much I've worked on each project instead of simply time spent working overall. Options for different timescales, options to color code each projects timer and stat tracker. More info so I can dissect my work ethic, etc.

2. Smoother sound level control! Jesus christ if you do nothing else on this list please do this. It's either blaring loud or simply off. I have to work in my room (1 bedroom, work from home) and I'm more productive at night and I always wake my girlfriend up because you have to set the sound level for each tomato and I forget sometimes.

3. Options to maybe change the length of your tomatos and breaks, and set up different patterns.

4. Integration with google calendar or maybe a project management app?

4. Maybe add more advanced project management features directly to tomato.es?

Sorry but tomato.es just isn't worth paying for in it's current state. If you want to have paid users you have to offer a service that no one else is offering (or at least a competitive service), and right now there are a million other free tools users can use.  I love tomato.es interface, which is why I use it, but there is a lot that can be done to improve the app, and make it worth paying for. Until then, I would just switch to another basic time tracker app. Just my 2c, and I hope you do what you think is best!

**Comment by potomak** [comment]
I'd like to thank you all for your feedback! 🙇

We're collecting a log of useful information. I like in particular how this thread is stimulating the conversation on a _controversial_ topic. It's also very motivating for me to see all these people expressing their ideas.

It seems like the majority of people who participated in this thread would stop using Tomatoes if we'd start charging for the service. It makes sense. Why should _we_ pay for Tomatoes when we can find alternative services that are free?

On the other side two more people started supporting Tomatoes by donating $8/month. That's almost a 23% increase in two days. Thank you. 🙇 

I think I'll remove that _annoying_ announcement box by the end of the week. After that I'll take some some time, let's say a couple of days, to think about what to do next. The outcome of this process will be a final proposal and, if needed, a timeline to make it happen. Your feedback is decisive for me to delineate a more clear picture of Tomatoes user base ideas, so please continue to share your opinion.

**Comment by metalelf0** [comment]
The app is good, it works, it's clean and simple but 4 USD per month is completely off-market. Think about it: Trello gold is 5 bucks per month. Of course it's a different app but it comes with a ton of features, integrations and it's way more mature. There are many free timers and in all honesty I think your app should offer more features to justify being paid. Some ideas? Well, Trello integration à-la Pomello. Anyway keep up the great work!

**Comment by babaliba** [comment]
First allow me to thank you for your explanation and for asking the users for our feedback. The fact that you can do that is the base for my main comment:

- The one thing you have got in your system is the “community”. You are actually asking the community for feedback and we are responding.

I have worked in several projects adding gamification for sales and directives and I realize that I have stopped using your system like “**_zenkamal_**” only when the leaderboards stopped working and I came back again when I realized that there were back again. Three things about this:

- Gamification would work in your site. Just a few budges and some sense of community could wonders. You have the user base!
- For the comments I saw the core users are freelance or people willing to work in their time management skills. We are more productive and we should pay for that.
- Those just wanting to use the site for a few tomatoes here and there away from the leaderboards or new elements should be able to do so. The community should support people testing this wonderful Pomodoro Method. Just let the “core” users have a premium service and let those who will never pay for anything lurk and feel welcomed.

In terms of pricing I used other Pomodoro apps when the statistics where not working and there are other cheap / free / good Pomodoro implementations. We are a global community, rich countries, poor countries, students, millionaires…. $4 a month is a lot for a lot of people and you have user forking out more than that and I would imagine they are happy about it….

- You should make the system SaaS but be flexible with price (I do not know how to do it but it is an idea)
- There are people like me that find a recurring monthly bill unappealing. I would prefer a yearly payment that I could write of easily like a company expense (12 receipts per year is messy)
- I like the idea of paying you and contributors for your work or paying the app infrastructure. I am not happy with donations to charity or non-profit. Again we are a global multicultural community, our link is your great app I cannot see everyone agreeing where to donate money.

**Comment by mariogintili** [comment]
I'm a user of the app, startup founder and ex consultant, my thoughts on this are:

- Happy to pay as long as the product doesn't get extended: My problem with most SaaS businesses is that they do one thing which I love and then end up doing 10 which I really don't care about
- Pricing seems a bit too expensive, $1 a month seems like the best fit.

**Comment by zenkamal** [comment]
Is there any way you might consider using something other than Backerpass, like Patreon?

Backerpass didn't have an option to change the backed amount, a way to log in, an "about us" section, etc etc. 

That made me a tiny little bit hesitant to back the project.

**Comment by potomak** [comment]
@zenkamal I was evaluating other services, such as https://opencollective.com/tomatoes, but there's no easy way to migrate and Backerpass was _good enough_ for the task.

**Comment by Liarra** [comment]
> Almost all of the the people that requested new features or bug fixing never actively helped by writing any code. Nobody has never even tried to host their own version of the app from what I know.

Aaand here comes my portion of shame. I've been intending to do so for a while now, because I love tomato.es and I think I could contribute to it as a UXer, while learning some Ruby meanwhile. If you are pulling it out of open source, it would be nice to get an early warning so I could clone the latest version of the project and have some hopes of returning to it. Eventually.

I would be fine to pay some symbolic amount of money to cover the hosting price. Maybe something like Wikipedia's fundraising banners that appear once every half year could work.

**Comment by andrew-chiz-muzychuk** [comment]
@potomak ,
Have you decided how you can get fund/to be donated?

**Comment by potomak** [comment]
@andrew-chiz-muzychuk not yet. The most natural choice seems to be Open Collective, but I didn't start the migration yet.

Do you have any feedback on the topic?

**Comment by tomcatt360** [comment]
Edit: I saw @yoX64 's comment and thought I should clarify something. I starting writing this blurb, then saw that the clarification had had already been made. Since I already wrote it though, here it is. Enjoy!

> * SaaS vs open source it's your choice, it does not depend so much on others.
> * I recommend open source, because people will be more happy to use it this way.

@yoX64 It's my understanding that @potomak is not making the project closed source, but that he's (potentially) making his implementation SaaS. The site "potato.es" would be on a subscription basis, but the code would still be kept open.

@potomak Correct me if I'm wrong. ;)
